tools/image_processing/yolo/yolo_training.xml

```diff
@@ -13,11 +13,19 @@
       mkdir -p ./input_images ./input_yolo ./training ./runs ./models output_files && 
 
       #for $filename in $input_images:
-          ln -s '$filename' './input_images/${filename.element_identifier}' &&
+          #if $filename.element_identifier.endswith($filename.ext)
```

Comment:
Is it a problem for yolo when the filename ends with .jgp.jpg?

Reply:
I just run it with file names like this, and they still work..
I wasn't sure initially, so just did a test run.

Reply:
Ok if it works you can simplify this and remove the if condition altogether or?

Reply:
I got a question, i'm a bit confused...
my tests failed after removing the if else..
if input image is` image1.jpg.jpg` then the input yolo txt will need to be `image1.jpg.txt` ..

will this work in a workflow, because the previous tool outputs the yolo txt, before yolo_training, does not have file extension?

I will try tomorrow.. thanks